### DISCUSSION TITLE
No ARMv7 SD Card Image?

### DISCUSSION THREAD

**Discussion by ShockWaveFlash** [awawa-dev/HyperHDR#511] [Q&A] [answered]
Hey

is there no v19 ARMV7 SD IMAGE for the Raspberry Pi 4?
From version 18 there was still

Thanks :)

**Comment by awawa-dev** [comment, marked as answer]
Hi
Raspberry Pi OS is and always was only in two version: armv6l and aarch64. Previously armv7 SD image was in fact armv6l OS with HyperHDR armv7 version installed on top. If you really want you can install armv6l SD image, remove HyperHDR (BTW now it's connected to APT repo) and manually install HyperHDR armv7 from Github Releases section. But it's not worth doing: the benefits will be minimal and you'll get a much bigger gain using the aarch64 SD image.

The manual is already updated.
https://github.com/awawa-dev/HyperHDR/wiki/Installation#raspberrypi-os-sd-card-image-installation
